packages/roc/2023.2.15/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+wget "https://github.com/roc-lang/roc/releases/download/nightly/roc_nightly-linux_x86_64-2023-02-15-b054977.tar.gz" -O "roc.tar.gz"
```

Comment:
![image](https://user-images.githubusercontent.com/45527309/219785166-0936b25d-3089-4690-9e55-84bfc03f674d.png)

`wget` does not work. use curl instead

Reply:
THANK YOU